guides/release/models/pushing-records-into-the-store.md

```diff
@@ -82,20 +85,23 @@ serializer before pushing it into the store, you can use the
 
 ```javascript {data-filename=app/serializers/album.js}
 import DS from 'ember-data';
+const { RESTSerializer } = DS;
 
-export default DS.RESTSerializer.extend({
+export default class AlbumSerializer extends RESTSerializer {
   normalize(typeHash, hash) {
     hash['songCount'] = hash['song_count']
     delete hash['song_count']
-    return this._super(typeHash, hash);
+    return super(typeHash, hash);
```

Comment:
elsewhere it seems like `...arguments` is used. Why are we doing things differently here?